Multisig/UI/App/UIKit/TransactionDetailsViewController.swift

```diff
@@ -45,48 +47,112 @@ class TransactionDetailsViewController: LoadableViewController, UITableViewDataS
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        title = "Transaction Details"
+
         builder = TransactionDetailCellBuilder(vc: self, tableView: tableView)
 
         tableView.delegate = self
         tableView.dataSource = self
 
         tableView.rowHeight = UITableView.automaticDimension
         tableView.estimatedRowHeight = 48
+
+        configureConfirmButton()
+
+        notificationCenter.addObserver(
+            self, selector: #selector(lazyReloadData), name: .ownerKeyRemoved, object: nil)
+        notificationCenter.addObserver(
+            self, selector: #selector(lazyReloadData), name: .ownerKeyImported, object: nil)
     }
 
-    override func reloadData() {
-        super.reloadData()
-        reloadDataTask?.cancel()
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        trackEvent(.transactionsDetails)
+    }
+
+    // MARK: - Signing
+
+    fileprivate func configureConfirmButton() {
```

Comment:
Can't we do this in the XIB?

Reply:
Maybe as a refactoring, but really no because this controller is instantiated from the LoadableViewController.xib, so we could load the button from XIB but we would still have to add constraints somewhere